automated install/basic-install.sh

```diff
@@ -1422,15 +1428,26 @@ install_manpage() {
     if [[ ! -d "/usr/local/share/man/man8" ]]; then
         # if not present, create man8 directory
         mkdir /usr/local/share/man/man8
+        chown root:staff /usr/local/share/man/man8
+        chmod a+r /usr/local/share/man/man8
+        chmod a+x /usr/local/share/man/man8
+        chmod g+s /usr/local/share/man/man8
```

Comment:
It should be sufficient to use the following permissions for man folders:
```
drwxr-xr-x. 2 root root 4096 Jul 12  2018 man1
```

Reply:
In `/usr/share/man` I have the same permissions as you, but in `/usr/local/share/man` the setgid bit is also set. Will this cause any trouble, if it is not set by the installer? (I am running Raspbian GNU/Linux 9 (stretch))
```bash
ls -la /usr/local/share/man
drwxrwsr-x 4 root staff 4096 Mär 30 14:42 .
drwxrwsr-x 8 root staff 4096 Nov 13 14:28 ..
drwxr-sr-x 2 root staff 4096 Mär 30 14:42 man5
drwxr-sr-x 2 root staff 4096 Apr  7 16:26 man8
```

```bash
ls -la /usr/share/man
drwxr-xr-x  34 root root  4096 Nov 13 14:22 .
drwxr-xr-x 227 root root 12288 Mär 30 14:37 ..
drwxr-xr-x   2 root root 69632 Apr  7 16:26 man1
drwxr-xr-x   2 root root 20480 Apr  7 16:26 man5
drwxr-xr-x   2 root root 36864 Mär 30 14:37 man8

```

Reply:
I don't think it would be an issue. Does it work fine on Raspbian?

Reply:
`man pihole` works in both cases and I did not find any information about the neccessity of setgid or group `staff` in the man pages of `man`, `mandb` or `manpath`, so I think it will be fine to leave it this way. Since `/usr/local/share/man` already has the setgid, the directories `man5` and `man8` should be created with group `staff` anyway, if I understand setgid correctly (I am not really experienced with linux) and there are no further directories inside anyway, aren't they?

Reply:
Right, it should only contain files.